### ISSUE TITLE
Invisible Mason (CUS issue)

### ISSUE THREAD

**Issue by GoogleFrog** [ZeroK-RTS/Zero-K#2966]
https://steamcommunity.com/sharedfiles/filedetails/?id=1371345277

**Comment by GoogleFrog** [comment]
https://pastebin.com/e996ZhyV

**Comment by sprunk** [comment]
```

[f=-000001] Custom Unit Shaders:, ERROR: 0:87: '' : syntax error: incorrect preprocessor directive
ERROR: 0:91: '' : syntax error: incorrect preprocessor directive
ERROR: 0:92: 'specular' : redefinition
ERROR: 0:93: 'reflection' : redefinition
ERROR: 0:120: '' : syntax error: incorrect preprocessor directive
 
 
[f=-000001] Custom Unit Shaders:, WARNING: 0:3: 'precision' : symbol not available in current GLSL version
WARNING: 0:3: 'highp' : symbol not available in current GLSL version
WARNING: 0:81: 'texture' : function is not available in current GLSL version
WARNING: 0:83: 'texture' : function is not available in current GLSL version
ERROR: 0:86: '' : syntax error: incorrect preprocessor directive
WARNING: 0:87: 'texture' : function is not available in current GLSL version
WARNING: 0:88: 'texture' : function is not available in current GLSL version
ERROR: 0:90: '' : syntax error: incorrect preprocessor directive
ERROR: 0:91: 'specular' : redefinition
ERROR: 0:92: 'reflection' : redefinition
ERROR: 0:119: '' : syntax error: incorrect preprocessor directive
 
 
[f=-000001] Custom Unit Shaders:, ERROR: 0:89: '' : syntax error: incorrect preprocessor directive
ERROR: 0:93: '' : syntax error: incorrect preprocessor directive
ERROR: 0:94: 'specular' : redefinition
ERROR: 0:95: 'reflection' : redefinition
ERROR: 0:122: '' : syntax error: incorrect preprocessor directive
 
 
[f=-000001] Custom Unit Shaders:, ERROR: 0:89: '' : syntax error: incorrect preprocessor directive
ERROR: 0:93: '' : syntax error: incorrect preprocessor directive
ERROR: 0:94: 'specular' : redefinition
ERROR: 0:95: 'reflection' : redefinition
ERROR: 0:122: '' : syntax error: incorrect preprocessor directive
 
 
[f=-000001] Custom Unit Shaders:, ERROR: 0:88: '' : syntax error: incorrect preprocessor directive
ERROR: 0:92: '' : syntax error: incorrect preprocessor directive
ERROR: 0:93: 'specular' : redefinition
ERROR: 0:94: 'reflection' : redefinition
ERROR: 0:121: '' : syntax error: incorrect preprocessor directive
```

**Comment by sprunk** [comment]
These
```
WARNING: 0:3: 'precision' : symbol not available in current GLSL version
WARNING: 0:3: 'highp' : symbol not available in current GLSL version
WARNING: 0:81: 'texture' : function is not available in current GLSL version
```
should've been fixed by 32d3ae4837e3b8cbf1c03e1176627e5df4ac99b5, the remaining ones are a duplicate of #2377
